webknossos/webknossos/dataset/_utils/pims_images.py

```diff
@@ -334,3 +348,23 @@ def dimwise_max(vectors: Sequence[T]) -> T:
         return vectors[0]
     else:
         return cast(T, tuple(map(max, *vectors)))
+
+
+C = TypeVar("C", bound=Type)
+
+
+def _recursive_subclasses(cls: C) -> List[C]:
+    "Return all subclasses (and their subclasses, etc.)."
+    # Source: http://stackoverflow.com/a/3862957/1221924
```

Comment:
do we need to worry about licensing?

Reply:
I don't think so, I copied this from pims, which is also licensed permissively. Also, I'd assume this is too simple to be copyrightable :tm: